### ISSUE TITLE
Multipoint annotations should be accessible to VoiceOver

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-gl-native#4838]
Point annotations are now accessible to VoiceOver, but multipoints (polylines and polygons) are not. Multipoints have no callout views, but it should still be possible to summarize them as, for instance, “{title}, northeast to southwest”.

/cc @friedbunny

**Comment by 1ec5** [comment]
This is blocked on a runtime layer API (#4839), rewriting `AnnotationManager` based on a style layer inserted at runtime, and feature eventing (#352).

**Comment by 1ec5** [comment]
#352 and #5200 have landed, so this issue is now blocked by #5165.
